### ISSUE TITLE
After Update to 0.8.22 - no messages there

### ISSUE THREAD

**Issue by ds2k5** [vector-im/riot-android#2930]
Hello,
after Update to 0.8.22 had the issue that old and new messages not show...
got notification that a new message is there... but can not see it...

here some screenshots, hope this helps to understand what I mean

https://www.bilder-upload.eu/bild-ce345f-1549609360.png.html
https://www.bilder-upload.eu/bild-f74993-1549609372.png.html
https://www.bilder-upload.eu/bild-478289-1549609387.png.html
https://www.bilder-upload.eu/bild-f9b0a6-1549609401.png.html
https://www.bilder-upload.eu/bild-76c360-1549609414.png.html

**Comment by ds2k5** [comment]
Sailfish OS 3.0.1.11 with Andorid-Support

**Comment by bmarty** [comment]
Ok. I don't know what it means in term of Android version...
If you have the opportunity to use the last version of riot from Jenkins, the bug should be fixed.
Or please wait for official release.

**Comment by ds2k5** [comment]
>Or please wait for official release.

Thats what in "Store" is offer
OK will ask Sailfish Support.
Thanks

**Comment by ivilata** [comment]
@bmarty, @ds2k5 seems to be using a phone with [SailfishOS](https://sailfishos.org/), which is not Android but includes an [Android compatibility layer](https://sailfishos.org/wiki/Android_Compatibility). The Android API level supported by the layer depends on the device SailfishOS is running on. According to [this answer](https://together.jolla.com/question/171676/some-android-apps-are-incompatible-with-sailfishx-why/?answer=171754#post-id-171754) (from some developer I guess, though I found other answers in other threads stating similar info), the Jolla 1 phone seems to support Android 4.1.2 (Jelly Bean), the Jolla C and Xperia X support Android 4.4.4 (Kit Kat).

I'm also using a Jolla 1 and experienced the same "blank message list" as @ds2k5 when I upgraded to Riot 0.8.22 from F-Droid. I was able to downgrade 0.8.21 and I got the messages back (and nothing seemed to break in the operation).
